### ISSUE TITLE
section('Aggregator'....) command crashes my python console

### ISSUE THREAD

**Issue by nicclemett** [zhuminjie/OpenSeesPyDoc#141]
Hi, 
**What I am trying to do:**
I am attempting to create an aggregated section with uncoupled Mz, My, Vz, and Vy force deformation relationships for use in the plastic hinge zone of a forceBeamColumn element.

**What the problem is**
When the section('Aggregator'...) command is called my script file terminates with no warnings or any indication of an error. No variables are shown in the Spyder variable explorer and any variables defined earlier in the script are deleted. The IPython console is reset to line[1]
Any new variables created do no appear in the variable explorer but can be accessed.

**What reproduces the issue**
I have a copy of the code that reproduces the issue.
below is also a screenshot showing the empty variable explorer and the output (none) in the IPython console (even with 'print' statements place in the code prior to the 'Aggregator' command.
In the provided code, if you comment out the 'Aggregator' commands and uncomment the second 'beamIntegration' command the code runs fine.

I can work around the problem by defining more elements between the nodes. each with their own material, however, this is not desirable.
I would appreciate any insights/help that you can provide.

Thanks!

![image](https://user-images.githubusercontent.com/62114051/76558160-1436cc00-649d-11ea-95ed-4434da0bf544.png)

[github_query_opsSectionAggregator.txt](https://github.com/zhuminjie/OpenSeesPyDoc/files/4325987/github_query_opsSectionAggregator.txt)

**Comment by mhscott** [comment]
Can you use *mats instead of mats in the command in order to dereference the lists?

--
Michael H. Scott, Ph.D.
Professor, Structural Engineering
Oregon State University
cce.oregonstate.edu/scott<https://exmail.oregonstate.edu/owa/redir.aspx?C=_F2i-PfJ4o6PTa3_ZQo4PE-QjaGSDCPQO_2ZTY0wP-CV6RK6qknXCA..&URL=https%3a%2f%2fcce.oregonstate.edu%2fscott>
OpenSees blog: portwooddigital.com<https://exmail.oregonstate.edu/owa/redir.aspx?C=RsSqkQx_bVLdtoYw647MW5oapuJIB618wYXBdJ4ymaiV6RK6qknXCA..&URL=http%3a%2f%2fwww.portwooddigital.com%2f>
________________________________

**Comment by nicclemett** [comment]
That worked! Thanks.
I should have read the docs better...

**Comment by nicclemett** [comment]
An additional comment...
Your suggestion above fixed the occurrence of the problem in the simple script I provided, however, when I added the asterisks to my analysis script I still encountered the same issue.

When comparing the example script (working) and my analysis script (not working) I noticed that the material tags in the example script were type 'int' and in the analysis script they were type 'numpy.int64' (I think because they were stored in and extracted from a pandas dataframe...?).

After using int() to convert the 'numpy.int64' to 'int' my analysis script ran with no issues.

Does OpenSeesPy have some issues taking 'numpy.int64' datatypes? I think I have encountered this issue before as well, though usually with a warning saying something along the lines of the datatype not being supported.
